ui-manchette/src/stories/ManchetteSplit.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import '@osrd-project/ui-core/dist/theme.css';
+import '@osrd-project/ui-manchette/dist/theme.css';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { SAMPLE_WAYPOINTS } from './assets/sampleData';
+import ManchetteSplit from '../components/ManchetteSplit';
+import TrackOccupancyManchette from '../components/TrackOccupancyManchette';
+import { type Waypoint } from '../types';
+
+const meta: Meta<typeof ManchetteSplit> = {
+  component: ManchetteSplit,
+  title: 'Manchette/ManchetteSplit',
+  tags: ['autodocs'],
+  argTypes: { content: { control: { type: 'object' } } },
+};
+
+export default meta;
+type Story = StoryObj<typeof ManchetteSplit>;
+
+const customDiv = (
+  <div style={{ height: 'auto', minHeight: 24, backgroundColor: 'rgba(152, 192, 245, 1)' }}>
+    Hello World
+  </div>
+);
+
+const gov = <TrackOccupancyManchette tracks={[{ id: '1', name: 'Track 1', line: 'A' }]} />;
+
+let reactNodeToInsert = [
+  { index: 1, reactNode: gov },
+  { index: 2, reactNode: customDiv },
+  { index: 3, reactNode: customDiv },
+  { index: 4, reactNode: customDiv },
+];
+
+const insertNodes = (
+  acc: (Waypoint | React.ReactNode)[],
+  nodes: { index: number; reactNode: React.ReactNode }[],
+  index: number
+) => {
+  nodes
+    .filter((item) => item.index === index)
+    .forEach((item) => {
+      acc.push(item.reactNode);
+      reactNodeToInsert = reactNodeToInsert.filter((node) => node.index !== item.index);
+    });
+};
+
+export const Default: Story = {
```

Comment:
It actually took me quite a while to understand what you were trying to achieve here... I think all these arrays manipulations serve no purpose here, but are distracting.

I prefer a way simpler version, that precisely shows what kind of inputs the component accepts, such as:

```typescript
export const Default: Story = {
  args: {
    content: [
      SAMPLE_WAYPOINTS[0],
      gov,
      SAMPLE_WAYPOINTS[1],
      customDiv,
      SAMPLE_WAYPOINTS[2],
      customDiv,
      SAMPLE_WAYPOINTS[3],
      customDiv,
    ],
  },
};
```